### DISCUSSION TITLE
server sent events

### DISCUSSION THREAD

**Discussion by crypt0miester** [Neoteroi/BlackSheep#268] [Q&A]
hi, so is there an implementation on sse? 

sorry I'm new to blacksheep.

**Comment by RobertoPrevato** [comment]
Hi @crypt0miester 
Thanks for your interest in the project. Unfortunately this feature is still missing from the framework.
However, the framework [supports WebSocket](https://www.neoteroi.dev/blacksheep/websocket/), which might be a good fit to cover the same use cases of SSE.

**Reply by crypt0miester** [reply]
excellent. I think I will try to fork https://github.com/sysid/sse-starlette and see if I can get it to work.

love your work. keep it up champ.
